lib/FrontendTool/FrontendTool.cpp

```diff
@@ -973,6 +973,34 @@ static Optional<bool> emitSILAfterSILGen(CompilerInvocation &Invocation,
                   opts.EmitSortedSIL);
 }
 
+static bool serializeMSF(FrontendOptions &opts, SILModule *SM,
```

Comment:
Please don't introduce a new acronym here. This is `serializeSIB` as written (note the serialization options that are turned on).

Reply:
Fixed.